### PR TITLE
Kv session

### PR DESCRIPTION
**Pull request by kvinwang**
This PR moves the `kv-session` implementation from the repo [phat-offchain-rollup](https://github.com/Phala-Network/phat-offchain-rollup) to here.

And merges the queue session and base session into one.

See the [e2e tests](https://github.com/Phala-Network/phala-blockchain/pull/1084/files#diff-ea8faa79710eebdee9e3f31f8f78b0f99954e9f64f6fd82dba08f0c467037a52R535) in anchor.rs for usage.

The crate has been published to https://crates.io/crates/pink-kv-session.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -69,6 +69,7 @@ members = [
 	"crates/pink-libs/s3",
 	"crates/pink-libs/utils",
 	"crates/pink-libs/subrpc",
+	"crates/pink-libs/kv-session",
```
Comment (h4x3rotab) [inline Cargo.toml:72]: Since it's a member, should we exclude crates/pink-libs/kv-session/Cago.toml?

**Review by h4x3rotab — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (kvinwang) [reply Cargo.toml:72]: Do you mean the `Cargo.lock`?
Yes, let me remove it.

Comment (kvinwang) [reply Cargo.toml:72]: Removed

**Pull request merged**